tests/integration/targets/rds_instance/main.yml

```diff
@@ -6,6 +6,5 @@
 - hosts: all
   gather_facts: no
   strategy: free
-  serial: 9
```

Comment:
Why are you removing this? It was 9 before to match the number of hosts.

Reply:
AFAIK `serial` defaults to all hosts, so if that is true it would be redundant

Reply:
Yeah, you're right :) I think I conflated the default for --forks with serial.